src/protos/io/defang/v1/fabric.proto

```diff
@@ -271,7 +271,6 @@ message ProjectUpdate {
   repeated ServiceInfo services = 1;
   string alb_arn = 2;
   string project = 3;
-  Behavior behavior = 4;
```

Comment:
I'm assuming this is removed as you shouldn't be able to change project behavior once already deployed?

Reply:
This was removed because it wasn't useful. Our pulumi code set the deployment mode in a global const (as well as global booleans `IS_PROD` and `IS_DEV`), so by the time we interpret this protobuf, those consts are already set and can't be changed. 

You must be able to change deployment modes from one deployment to another (in fact, that's the only way completely remove all storage for a service, since a production deployment can never destroy storage, so the only workaround is to redeploy as development or staging, and destroy/down that.)